src/main/java/seedu/address/ui/BrowserPanel.java

```diff
@@ -68,6 +73,19 @@ public void freeResources() {
     @Subscribe
     private void handlePersonPanelSelectionChangedEvent(PersonPanelSelectionChangedEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
-        loadPersonPage(event.getNewSelection().person);
+        // loadPersonPage(event.getNewSelection().person);
+        ReadOnlyPerson rop = event.getNewSelection().person;
```

Comment:
A little stupid, but can we name this variable `person` just to be consistent with the rest of the project?